web/app.fluidity.money/app/routes/$network/dashboard/rewards/index.tsx

```diff
@@ -73,11 +73,11 @@ export const loader: LoaderFunction = async ({ request, params }) => {
       count,
     }: { transactions: UserTransaction[]; count: number } = await (
       await fetch(
```

Comment:
Not going to block on this but maybe we should consider moving this into a function or hook

Reply:
Will do in a sep branch